config/config.exs

```diff
@@ -20,6 +20,15 @@ config :subwayside_ui, SubwaysideUiWeb.Endpoint,
   pubsub_server: SubwaysideUi.PubSub,
   live_view: [signing_salt: "Gjj0V7iV"]
 
+config :subwayside_ui, SubwaysideUi.GTFS.TrainCrowdingStatus,
+  feed_func: {SubwaysideUi.GTFS.TrainCrowdingStatus, :get_gtfs_realtime_feed, []}
+
+config :subwayside_ui, SubwaysideUi.GTFS,
+  # Which GTFS-RT (JSON) feed to load:
+  url: "https://mbta-gtfs-s3.s3.amazonaws.com/rtr/VehiclePositions_enhanced.json",
+  # Hide trains that don't have at least one car in GTFS-RT feed:
+  only_show_gtfs: true
```

Comment:
This feature flag is a result of a conversation with @fsaid90 - we only want to show trains that exist in the GTFS-RT payload. It works by ensuring at least one of the cars has occupancy data before showing the entire train.